Fabric.Authorization.API/scripts/run-client-functional-tests.sh

```diff
@@ -23,21 +37,33 @@ echo "started identity"
 sleep 3
 
 curl -sSL https://raw.githubusercontent.com/HealthCatalyst/Fabric.Identity/master/Fabric.Identity.API/scripts/setup-samples.sh > identity-setup-samples.sh
-output=$(. identity-setup-samples.sh)
+output=$(. identity-setup-samples.sh "http://localhost:5001")
 installerSecret=$(echo $output | grep -oP '(?<="installerSecret":")[^"]*')
 authClientSecret=$(echo $output | grep -oP '(?<="authClientSecret":")[^"]*')
 
 export FABRIC_INSTALLER_SECRET=$installerSecret
 echo $installerSecret
 echo $authClientSecret
 
+if [ "$IMAGE" -eq "1" ]; then
+	echo "Using docker image"
+	IMAGE_NAME="healthcatalyst/fabric.authorization"
+else
+	echo "Using local code"
+	IMAGE_NAME="authorization.functional.api"
+	cd ..
+	dotnet publish -o obj/Docker/publish
+	docker build -t authorization.functional.api .		
```

Comment:
Could we replace the image name in the `docker build` command with `$IMAGE_NAME`

Reply:
right on